kuma/core/utils.py

```diff
@@ -321,3 +326,15 @@ def get_unique(content_type, object_pk, name=None, request=None,
     unique_hash = hashlib.md5(hash_text).hexdigest()
 
     return (user, ip, user_agent, unique_hash)
+
+
+def bitly_shorten(url):
+    if not settings.USE_SHORT_URLS:
+        return None
+
+    try:
+        return bitly.shorten(url)['url']
+    except (bitly_api.BitlyError, KeyError) as e:
+        # If bit.ly service fails or the API key isn't configured.
+        log.error('Bitly URL shortening failed: %s' % e)
```

Comment:
I think we should use `log.exception` here (this was a bug in the previous code as well). Without it we don't get the traceback object passed to the logger.